src/main/java/com/smartbear/readyapi/plugin/git/ui/AbstractRepositorySelectionGui.java

```diff
@@ -36,9 +37,12 @@ protected Component createGui(String helpUrl, String helpText, String remoteRepo
         panel.add(sshRadioButton);
         JRadioButton httpsRadioButton = createRadioButton(LABEL_HTTPS, group);
         panel.add(httpsRadioButton);
+        JRadioButton localRadioButton = createRadioButton("Local", group);
```

Comment:
The "Local" value is used two times. It is better to use a variable instead of a value.

Reply:
+1

Reply:
Fixed